lib/web_console/exception_mapper.rb

```diff
@@ -50,7 +50,7 @@ def guess_the_first_application_binding
         @bindings.find do |binding|
           source_location = SourceLocation.new(binding)
           source_location.path.to_s.start_with?(Rails.root.to_s)
-        end
+        end if defined?(Rails.root) && Rails.root
```

Comment:
I don't think this change should be made like this. We should be configuring the application path, and using the Railtie we can set it to `Rails.root`

Reply:
`Rails.root` is not a method in my case. Would you prefer just checking for the method

```suggestion
        end if defined?(Rails.root)
```

Reply:
No. I'd prefer if this class didn't call `Rails.root` at all, and instead had a `application_root` configuration that could be set by this gem `Railtie` to `ExceptionMapper.application_root = Rails.root`